.gitlab-ci.yml

```diff
@@ -6,11 +6,6 @@ clippy:
   image: paritytech/ci-linux:105b919b-20210617
   stage: test
   timeout: 1 hours
-  only:
-    refs:
-      - branches
-    changes:
-      - ./**/*.{rs,toml,lock}
   script:
     - rustup component add clippy --toolchain nightly
     - cargo +nightly clippy -- -D warnings
```

Comment:
Would it make sense to add `--all-features --all-targets` as well? At least for clippy, `--all-targets` detects a whole new class of warnings that are otherwise not caught.

Reply:
bfb0a36